cpu/samd21/cpu.c

```diff
@@ -22,6 +22,10 @@
 #include "periph_conf.h"
 #include "periph/init.h"
 
+#ifndef VDD_MILLIVOLTS
+#define VDD_MILLIVOLTS 3300
+#endif
```

Comment:
Sorry for being sloppy, I think the guarded #define should stay here as a fallback. In `periph_conf.h` an unguarded define but additional documentation would be great.

I hope that's more clear.